client/layout/color-scheme/index.jsx

```diff
@@ -1,3 +1,4 @@
+import config from '@automattic/calypso-config';
 import { getAdminColor } from 'calypso/state/admin-color/selectors';
 import { getPreference } from 'calypso/state/preferences/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
```

Comment:
I wish the preference helper still allowed us to change the color scheme in real time:


https://github.com/user-attachments/assets/285b2324-327e-483f-8c85-b09c8017df59

This is how it behaves in trunk, though - I guess it is intentional, to prioritize admin color scheme instead of the preference:

https://github.com/Automattic/wp-calypso/blob/6142a40a134417947d48edc471030a7488402cc6/client/layout/color-scheme/index.jsx#L16

Still, allowing the preference to work would be useful for development and quick changes of the color scheme without having to navigate to `/wp-admin/profile.php`. Maybe we could change that and just not persist that preference beyond the current session.

Reply:
I could add your request to https://github.com/Automattic/wp-calypso/issues/98335.

My priority with https://github.com/Automattic/wp-calypso/pull/98380 and this PR was mainly to remove browser console warnings, and I wasn't planning on iterating on the reaming wish list from #98335

Reply:
Sounds good, this isn't intended to be a blocker 👍

Reply:
Cool, updated https://github.com/Automattic/wp-calypso/issues/98335#issuecomment-2590252022